### ISSUE TITLE
Undefined lastUpdate causes errors

### ISSUE THREAD

**Issue by ahmadalfy** [unindented/storybook-addon-rtl#20]
I followed the instructions and the plugin works perfectly but there is an error in the console:

```
TypeError: Cannot read properties of undefined (reading 'userInteraction')
    at Object.<anonymous> (vendors~main.manager.bundle.js:32038)
    at vendors~main.manager.bundle.js:70619
    at Array.forEach (<anonymous>)
    at Channel.handleEvent (vendors~main.manager.bundle.js:70618)
    at PostmsgTransport.<anonymous> (vendors~main.manager.bundle.js:70503)
    at PostmsgTransport.handler (vendors~main.manager.bundle.js:49601)
    at PostmsgTransport.handleEvent (vendors~main.manager.bundle.js:49770)
error @ vendors~main.manager.bundle.js:72288
```

Specifically this function:

```javascript
  channel.on(_coreEvents.STORY_RENDERED, function (_) {
    var _channel$last;

    // lastUpdate in the following line will be undefined
    var lastUpdate = (_channel$last = channel.last(_constants.UPDATE_EVENT_ID)) === null || _channel$last === void 0 ? void 0 : _channel$last[0];
    lastUserInteractionValue = lastUpdate.userInteraction ? lastUpdate.direction : lastUserInteractionValue;
    var paramValue = api.getCurrentParameter('direction');
    var newDirection;

    if (paramValue) {
      newDirection = paramValue;
    } else if (lastUserInteractionValue) {
      newDirection = lastUserInteractionValue;
    } else {
      newDirection = (0, _utils.getDefaultTextDirection)(api);
    }

    channel.emit(_constants.UPDATE_EVENT_ID, {
      direction: newDirection,
      userInteraction: false
    });
  });
```

**Comment by literalpie** [comment]
Thanks for the report. I'll try to get a fix in soon

**Comment by literalpie** [comment]
A fix has been merged and released as version 0.4.2!
